III.tdd-katas/fizz-buzz/fizz-buzz-reimplementation.js

```diff
@@ -0,0 +1,27 @@
+module.exports = n => {
+  var result = ""; //This variable used to store the final result
+  var i = 1;
+  while (i <= n) {
```

Comment:
wwell doneNice :+1: 

one way to check if a number `X` is multiple of two **odd** numbers `a` and `b` is checking that
number `X` is multiple of `a * b`. For example, to find out if a number is multiple of 5 and 7 we can find if that number is multiple of 35 (5*7):

* 105 multiple of 35 and therefore it is also multiple of both 5 and 7
* 63 is not multiple of 35, therefore it is not a multiple of both 7 and 5

We could use this approach to improve our implementation